CHANGES.md

```diff
@@ -1,3 +1,6 @@
+# 2.5.0 UNRELEASED
```

Comment:
I think this content is just a patch release for a fix (i.e. 2.4.1). I'm happy for this to remain as `UNRELEASED` until we know the full content of the release, but I do see the value in setting it if we know that same change necessitates a particular version change.